canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -414,6 +415,18 @@ class VirtualizedTable extends React.Component {
 						readOnly={this.props.readOnly}
 					/>
 				</div>);
+			} else {
+				trashOption = (
+					<Button
+						kind="ghost"
+						size="sm"
+						type="delete"
+						className="delete-button"
+						hasIconOnly
+						onClick={this.props.deleteRow}
+						renderIcon={TrashCan}
```

Comment:
Hey Neha, I removed iconDescription because it would fail tests due to the fact that the icon description is considered to be part of the text. Let me know if I should just remove icon description or to modify the test cases. 

<img width="1141" alt="image" src="https://github.com/user-attachments/assets/4509899e-2a8b-449f-aa78-2457b929db84">

Reply:
@mikieyx You should add `iconDescription` and modify test cases.